http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/auth/SamAuthProvider.scala

```diff
@@ -312,6 +312,9 @@ class SamAuthProvider[F[_]: OpenTelemetryMetrics](
   override def isCustomAppAllowed(userEmail: WorkbenchEmail)(implicit ev: Ask[F, TraceId]): F[Boolean] =
     samDao.isGroupMembersOrAdmin(config.customAppCreationAllowedGroup, userEmail)
 
+  override def isSasAppAllowed(userEmail: WorkbenchEmail)(implicit ev: Ask[F, TraceId]): F[Boolean] =
```

Comment:
Naive question (and might be irrelevant with the consumption model), but should a Sam admin be able to access the license, or should only a group member be able to?

Reply:
I am referring to the `isGroupMembersOrAdmin` function from the samDAO below, would it give access to a test super user that is a Sam Admin, but not in the `sasAppCreationAllowedGroup`?